mr/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -82,11 +82,13 @@ public boolean needsTaskCommit(TaskAttemptContext context) {
 
   /**
    * Collects the generated data files and creates a commit file storing the data file list.
-   * @param context The job context
+   * @param ctx The task attempt context
```

Comment:
What is the value of renaming this to `ctx`? We generally prefer the longer names because they are easier to read and to type.

Reply:
I did this so we can keep the original references (`context` and `jobContext`) throughout the methods without renaming them in all places, since we replace the input context (`ctx`) with the enriched version right away. But I see your point, and I'm equally fine with keeping the original input name (`context`), and give a different name to the enriched references instead.

Reply:
Okay, I understand. Let's do this, but use a better name that doesn't violate our best practices. How about `originalContext` or `contextBeforeEnrichment` instead?

Reply:
Sounds good!